src/routes/e2e/widget/inline-instance/+page.svelte

```diff
@@ -11,6 +11,7 @@
 
   let formEl: HTMLDivElement;
   let userInfo: Record<string, string> | null;
+  // TODO: Use a more specific type
   let widget: any;
```

Comment:
These are all the e2e test routes, so I'm not all that worried about the typing here right now.